site_scons/site_tools/go_builder.py

```diff
@@ -81,6 +91,10 @@ def _check_go_version(context):
             context.Result(0)
             return 0
 
+        context.Display(f'Getting minimum {env.d_go_bin} version... ')
+        min_go_version = get_min_go_version()
```

Comment:
```suggestion
        min_go_version = get_min_go_version()
        if min_go_version is None:
            context.Result("no minimum go version found in go.mod")
            return 1
```
This will need testing and adjusting for whitespace I assume.

Reply:
Yeah, this was similar to what I had before. I'll change it back.

Reply:
I missed that, sorry, but I think this is the correct way to fail the build if the check cannot be run.